src/scss/colors.scss

```
@@ -0,0 +1,38 @@
+@use "sass:color";
+
+$colors-enum: (
+  primary: #7c46fb,
+);
+$shades-enum: (5, 10, 14, 20, 50);
+$alphas-enum: (0.18, 0.2, 0.24, 0.35, 0.5, 0.56, 0.65, 0.8);
+
+@function hexToRGB($hex) {
+  @return red($hex), green($hex), blue($hex);
+}
```

Snake case convention in (S)CSS.

```suggestion
@function hex-to-rgb($hex) {
  @return red($hex), green($hex), blue($hex);
}
```

This looks like an overkill for the white and black colors. 

The idea was a function can be used / created **if needed**, but one can write these colors by hand (not even online calculator required) 🤷

---

fixed in 8e20515ab04bf4df178d893dadb185d0e7b6126e

---

removed at all, passed as RGB format